static/main.css

```diff
@@ -71,3 +73,15 @@ ul {
 ul li button {
   min-width: 200px;
 }
+
+.ChatList {
+  width: 30%;
+  float: left;
+}
+
+.ChatView {
+  width: 70%;
+  float: right;
+  height: 800px;
```

Comment:
Maybe `height: 100%'?

Reply:
Height 100% makes it just as big as it needs to be with a scroll bar but not fitting the window height. There's some property i can get from electron, i will give it a try

Reply:
> I don't really want to continue "monkey patching/overwriting" css stuff outside of conversations.

Yet another reason to move conversations into desktop :)